src/Refactoring-Core/RBAddNewClassRefactoring.class.st

```diff
@@ -0,0 +1,34 @@
+"
+This refactorings differs from the Insert New Class refactoring in that does not reparent the subclasses to the new class.
+"
+Class {
+	#name : 'RBAddNewClassRefactoring',
+	#superclass : 'RBInsertNewClassRefactoring',
+	#category : 'Refactoring-Core-Refactorings',
+	#package : 'Refactoring-Core',
+	#tag : 'Refactorings'
+}
```

Comment:
The purpose of AddNewClassRefactoring is that does not reparent the subclass(es) to the new class.

I understand here you want to avoid having a new class? 

In this sense I'm not sure it is better "to be clever" (it is always a mess to check which message should be avoided to have one feature active or not. I would prefer to have two explicit refactoring classes: One for adding a class with reparent (Insert), and another one without reparent (just Add).

Reply:
I will make it as resolved for now, but feel free to open a new issue or we could check it later if it makes sense to merge both classes.

Reply:
Let's remove this class, since we have InsertNewClass, let's stay on track and follow the API we have so far